spec/index.bs

```diff
@@ -12961,7 +12961,8 @@ GPUQueue includes GPUObjectBase;
             <div data-timeline=device>
                 [=Device timeline=] steps:
 
-                1. If any of the following requirements are unmet, [$generate a validation error$] and stop.
+                1. If any of the following requirements are unmet, [$generate a validation error$], make each
+                    {{GPUCommandBuffer}} in |commandBuffers| [=invalid=], and stop.
```

Comment:
nit: I think you can just move the existing
```
                1. For each |commandBuffer| in |commandBuffers|:
                    1. Make |commandBuffer| [=invalid=].
```
step up above the validation.

Reply:
Wait, never mind.

Reply:
That would cause the validation steps to fail